src/accelerate/big_modeling.py

```diff
@@ -387,8 +387,12 @@ def dispatch_model(
         retie_parameters(model, tied_params)
     else:
         device = list(device_map.values())[0]
-        if device != "disk":
+        # for backward compatibility
+        is_quantized = getattr(model, "is_quantized", False) or getattr(model, "is_loaded_in_8bit", False)
+        if device != "disk" and not is_quantized:
             model.to(device)
+        elif is_quantized:
+            pass
         else:
```

Comment:
```suggestion
        elif not is_quantized:
```

Reply:
Ah added it already [b93d93c](https://github.com/huggingface/accelerate/pull/1652/commits/b93d93c412d43983b1021aee69d9e4683b54eff2) before your review :D